### ISSUE TITLE
Grid Selection not working in Firefox

### ISSUE THREAD

**Issue by vivek201** [telerik/kendo-angular#826]
**Only in Firefox (using the latest version 55.0.1 (64-bit))**

Plunker: [http://plnkr.co/edit/mn3tjD92LJ02cUsT59N9?p=preview](http://plnkr.co/edit/mn3tjD92LJ02cUsT59N9?p=preview)
This is just the sample plunker that was provided in the documentation

Clicking on the table cells to select the row causes the following error to pop up in the console.
```
ERROR ReferenceError: event is not defined
Stack trace:
a</e.prototype.onMousedown@http://www.telerik.com/kendo-angular-ui/npm/node_modules/@progress/kendo-angular-grid/dist/cdn/js/kendo-angular-grid.js:15:17678
decoratePreventDefault/<@https://unpkg.com/@angular/platform-browser@4.2.2//bundles/platform-browser.umd.js:2702:53
ZoneDelegate.prototype.invokeTask@https://unpkg.com/zone.js@0.8.11/dist/zone.js:424:17
onInvokeTask@https://unpkg.com/@angular/core@4.2.2//bundles/core.umd.js:3956:28
ZoneDelegate.prototype.invokeTask@https://unpkg.com/zone.js@0.8.11/dist/zone.js:423:17
Zone.prototype.runTask@https://unpkg.com/zone.js@0.8.11/dist/zone.js:191:28
ZoneTask/this.invoke@https://unpkg.com/zone.js@0.8.11/dist/zone.js:486:28
  core.umd.js:1052:5

```
Clicking the checkbox works (I think, I haven't checked the data from SelectionEvent yet).

**Comment by vivek201** [comment]
@tsvetomir I didn't know who else to tag. Can you look into it and see if there's anything I can do right now?

**Comment by rusev** [comment]
@vivek201 Indeed this is a bug on our side. We've been able to spot the cause of the issue. Unfortunately we cannot recommend a workaround. You will have to wait for the fix.

**Comment by vivek201** [comment]
@rusev Thanks for looking into it. Hope the fix comes out soon.

**Comment by siyka-marinova** [comment]
Fix for this bug is currently available in the latest dev. version: `@progress/kendo-angular-grid@1.3.2-dev.201708151057`

**Comment by tsvetomir** [comment]
Fixed in `@progress/kendo-angular-grid@1.3.2`.
